fredericoo/siggy-app
# feat: save company parameters on database

# Overview

Implementing the saving of company parameters on every change.

## What we have done

- Created a new column on the database to save the JSON of parameters. As they'll take all sorts of shapes, a JSON will work. It is then parsed and transformed into values. It shouldn't bee too unsafe.

## Blockers/Issues

- Code is getting complex and we should think of refactoring the tabs to their own organisms

## How to test

- Create a signature
- Change some company data
- Save it and see how it goes

## Checklist:
